RELEASES.md

```diff
@@ -129,14 +129,17 @@ for the list of actively tested versions. Kubernetes only supports n-3 minor
 release versions and containerd will ensure there is always a supported version
 of containerd for every supported version of Kubernetes.
 
-| Kubernetes Version | containerd Version | CRI Version  |
-|--------------------|--------------------|--------------|
-| 1.20               | 1.5.0+             | v1alpha2     |
-| 1.21               | 1.5.0+             | v1alpha2     |
-| 1.22               | 1.5.5+             | v1alpha2     |
-| 1.23               | 1.6.0+, 1,5.8+     | v1, v1alpha2 |
-| 1.24               | 1.6.4+, 1.5.11+    | v1, v1alpha2 |
-| 1.25               | 1.6.4+, 1.5.11+    | v1, v1alpha2 |
+| Kubernetes Version | containerd Version | CRI Version     |
+|--------------------|--------------------|-----------------|
+| 1.20               | 1.5.0+             | v1alpha2        |
+| 1.21               | 1.5.0+             | v1alpha2        |
+| 1.22               | 1.5.5+             | v1alpha2        |
+| 1.23               | 1.6.0+, 1,5.8+     | v1, v1alpha2    |
+| 1.24               | 1.6.4+, 1.5.11+    | v1, v1alpha2    |
+| 1.25               | 1.6.4+, 1.5.11+    | v1, v1alpha2    |
+| 1.26               | 1.6.tbd, 1.7.tbd   | v1, v1alpha2 ** |
```

Comment:
Maybe
```suggestion
| 1.26               | 1.6.tbd, 1.7.tbd   | v1    |
```
And move `**` to the k 1.25 line

Reply:
```
| 1.23               | 1.7.tbd, 1.6.0+, 1,5.8+     | v1, v1alpha2    |
| 1.24               | 1.7.tbd, 1.6.4+, 1.5.11+    | v1, v1alpha2    |
| 1.25               | 1.7.tbd, 1.6.4+, 1.5.11+    | v1, v1alpha2 ** |
| 1.26               | 1.6.tbd, 1.7.tbd                | v1                      |
```
Since v1.7 will support CRI v1 and v1alpha2, most likely it can support v1.23~v1.26. Needs more tests.
Will we add it to v1.23~v1.26?

Reply:
Updated.

Reply:
> Maybe
> 
> And move `**` to the k 1.25 line

Nod.. we can go with the context that this CRI column is support by the named k8s release. The extra CRI api dimension makes this table complex now that k8s has removed one. Another way to do it would be to put the CRI apis supported by k8s/containerd in different columns, making it a truth table.